src/BookingStateWrapper/MultiCityTrip.js

```diff
@@ -14,8 +15,9 @@ const MultiCityTrip = ({ children, booking }: Props) => (
   <BookingStateProvider
     hasBooking
     isPastBooking={booking.isPastBooking}
-    isUrgent={false /*fixme*/}
+    departureTime={idx(booking, _ => _.trips[0].departure.time)}
```

Comment:
why `BookingStateProvider` requires `departureTime` which is property derived from `booking` prop -> should be necessary to pass just booking and derive the rest from it.